cborg/src/Codec/CBOR/Read.hs

```diff
@@ -1796,7 +1796,7 @@ tryConsumeInteger hdr !bs = case word8ToWord hdr of
   0x1b -> let !w = eatTailWord64 bs
               sz = 9
 #if defined(ARCH_32bit)
-          in DecodedToken sz (BigIntToken (isWord64Canonical sz (word64ToWord w)) $! toInteger w)
+          in DecodedToken sz (BigIntToken (isWord64Canonical sz w) $! toInteger w)
```

Comment:
Also looks like this couldn't ever have been type correct.

Reply:
see https://github.com/well-typed/cborg/pull/322/files#r1320791126